.gitignore

```diff
@@ -33,3 +33,4 @@
 npm-debug.log
 testem.log
 /.chrome
+yarn.lock
```

Comment:
It is - (depends on the project) and then I would need to commit it to the Material 2 repository?